code/modules/tgui/modules/_base.dm

```diff
@@ -10,8 +10,8 @@ Code is pretty much ripped verbatim from nano modules, but with un-needed stuff
 	var/name
 	var/datum/host
 
-/datum/tgui_module/New(var/host)
-	src.host = host
+/datum/tgui_module/New(datum/_host)
+	host = _host
```

Comment:
I know src is bad. But this will just make the params a bit less standard so to say. Can we either make this the coding default or allow an exception for use of src in Inits/News? Quite some language standards make the exception here as well

Reply:
Not relevant to this pull request